ksql-engine/src/test/java/io/confluent/ksql/ddl/commands/CreateStreamCommandTest.java

```diff
@@ -134,6 +140,23 @@ public void shouldThrowOnOldWindowProperty() {
     createCmd();
   }
 
+  @Test
+  public void testCreateAlreadyRegisteredStreamThrowsException() {
+    final CreateStreamCommand cmd;
+
+    // Given:
+    givenProperties(propsWith(ImmutableMap.of()));
+    cmd = createCmd();
```

Comment:
```suggestion
    final CreateStreamCommand cmd = createCmd();
```

Reply:
What's wrong with the declaration at the beginning of the method? I think it looks cleaner, but is there a benefit on moving the declaration to this line?